blockchain/state_manager/src/lib.rs

```diff
@@ -153,28 +153,28 @@ where
         &self.chain_config
     }
 
-    /// Gets actor from given [Cid], if it exists.
+    /// Gets actor from given `[Cid]`, if it exists.
```

Comment:
These three things are different:

    [Cid]         -- Creates a link to Cid
    `[Cid]`       -- Code snippet containing "[Cid]"
    [`Cid`]       -- Code snippet that links to Cid

You want the last one. When in doubt, run `cargo doc --open` and make sure the documentation looks right.

Reply:
This should be fixed everywhere.